futures-util/src/stream/chunks.rs

```diff
@@ -112,3 +97,15 @@ impl<S> Stream for Chunks<S>
         }
     }
 }
+
+/* TODO
+// Forwarding impl of Sink from the underlying stream
+impl<S> Sink for Chunks<S>
+    where S: Sink + Stream
+{
+    type SinkItem = S::SinkItem;
+    type SinkError = S::SinkError;
+
+    delegate_sink!(stream);
+}
```

Comment:
Why are these sink forwarding impls sometimes commented out? If these can be enabled, I can provide a PR for this.

Reply:
They were all commented out because many of the `Stream` combinators were ported to 0.3 before the `Sink` trait had been. Feel free to re-enable as you wish.